docs/development/SETUP.md

```diff
@@ -70,6 +70,24 @@ VEDA-UI includes a GitHub action for checking TypeScript and lint errors. If you
 
 Additionally, there's a pre-commit hook that performs the same error checks. This helps developers identify and address issues at an earlier stage. If you need to bypass the check (to make a local temporary commit etc.), include the `--no-verify`` flag in your commit command.
 
+#### Format on Save
+While the existing checks ensure that no linting errors are introduced into the main branch, enabling "Format on Save" in your code editor provides an additional layer of convenience. It helps catch formatting issues early in the development process, minimizing the risk of committing errors and ensuring consistent code style across the team.
+
+##### How to Set Up Format on Save in VS Code
+Install Required Extensions:
+
+1. Install the ESLint extension ("dbaeumer.vscode-eslint").
+2. Enable Format on Save in the settings (<kbd>⌘ Command</kbd> + <kbd>,</kbd> on macOS)
+3. Configure Formatter Settings:
+   Ensure ESLint is set as the default formatter:
+   ```json
+   "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+   ```
```

Comment:
```suggestion
   Set ESLint as the default formatter:

   **Using Settings UI**: Open the Command Palette (Cmd+Shift+P or Ctrl+Shift+P), select Preferences: Open Settings (UI), search "default formatter," and select ESLint.

   **Using JSON**: Open Preferences: Open User Settings (JSON) and add:
   
   ```
   "editor.defaultFormatter": "dbaeumer.vscode-eslint"
   ```
```

Reply:
Thanks @sandrahoang686 ! Do you think adding more detailed instructions would help? I did not want to go too much into vs code specifics, as people may be using different editors. Maybe with the updated note below, we don't need to change anything here at all? Let me know what you think!

Reply:
yeah I think that helps! thanks for working on this & updating 🙇🏼‍♀️